desktop-app/src/renderer/components/DeviceManager/index.tsx

```diff
@@ -111,6 +123,50 @@ const DeviceManager = () => {
           <div className="flex w-fit flex-col items-start px-1">
             <h2 className="text-2xl font-bold">Manage Devices</h2>
           </div>
+          <div>
+            <DropDown
+              label={
+                <div className="flex items-center gap-2">
+                  <Icon icon="mdi:devices" fontSize={18} />
+                  <span>Device Type</span>
+                </div>
+              }
+              options={[
+                {
+                  label: (
+                    <div className="flex items-center gap-2">
+                      {filteredType === null && <Icon icon="mdi:check" />}
+                      <span
+                        className={cx('capitalize', {
+                          'font-semibold text-black dark:text-white':
+                            filteredType === null,
+                        })}
+                      >
+                        All Device Types
+                      </span>
+                    </div>
+                  ),
+                  onClick: () => setFilteredType(null),
+                },
+                ...deviceTypes.map((type) => ({
+                  label: (
+                    <div className="flex items-center gap-2">
+                      {filteredType === type && <Icon icon="mdi:check" />}
+                      <span
+                        className={cx('capitalize', {
+                          'font-semibold text-black dark:text-white':
+                            filteredType === type,
+                        })}
+                      >
+                        {type}
+                      </span>
+                    </div>
+                  ),
```

Comment:
This is the same as the lines 137 to 147, please refactor into a reusable separate component. You could call it DeviceManagerLabel if you'd like, and pass props to it to handle variations